docs/book/src/tasks/upgrading-clusters.md

```diff
@@ -0,0 +1,42 @@
+# Upgrading management and workload clusters
+
+## Considerations
+
+### Supported versions of Kubernetes
+
+If you are upgrading the version of Kubernetes for a cluster managed by Cluster API,
```

Comment:
Should we say something about not skipping minor versions = (eg. 1.16.x --> 1.18.x) ?

Reply:
Added